gradle/verification-metadata.xml

```diff
@@ -3092,9 +3102,14 @@
             <sha256 value="6ec357337b65082419e123db5e844df3135694b6891083884c02b1c11f301d6b" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.jetbrains.kotlin.android" name="org.jetbrains.kotlin.android.gradle.plugin" version="1.8.21">
+         <artifact name="org.jetbrains.kotlin.android.gradle.plugin-1.8.21.pom">
+            <sha256 value="85e4a6a46ce6d234d12421ef3865664714e8afbf7287a651aa580646821340a6" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.jetbrains.kotlin.plugin.serialization" name="org.jetbrains.kotlin.plugin.serialization.gradle.plugin" version="1.8.21">
          <artifact name="org.jetbrains.kotlin.plugin.serialization.gradle.plugin-1.8.21.pom">
-            <sha256 value="78403d230b0bcf54f94593dbd18267d2f4a275dc3ea44d2ec2cade26537a4baa" origin="Generated by Gradle"/>
+            <sha256 value="8426766c4866369ca9ac2c9ad94879059a34b72e0b6a8e20317fdb55f967fc26" origin="Generated by Gradle"/>
```

Comment:
I'm not sure why Gradle generates this for you, but this hash doesn't verify for me.